controllers/designate_controller.go

```diff
@@ -1339,6 +1389,127 @@ func (r *DesignateReconciler) generateServiceConfigMaps(
 	return nil
 }
 
+func (r *DesignateReconciler) getEnvVars(
```

Comment:
I'm not entirely sure what we are doing here... I think we might be getting tangled between the old way of setting a pile of configuration via a bash script and the other through templates..

Reply:
Yes, I totally agree. So after Brent and I went over the code, we decided we should have an additional hash, PoolsYamlHash, that won't be depended on the environment variables. I have just uploaded a new version.